src/components/contact-info-section/ContactInfoSection.tsx

```diff
@@ -13,13 +13,15 @@ interface ContactInfoSectionProps {
   inLine: boolean;
   changePatron: ChangePatronProps;
   showCheckboxes: boolean;
+  requiredFields?: ("email" | "phone")[];
```

Comment:
Personally I prefer pulling this into a separate type - even if it just a simple string literal.